config/k8s-installer/capi/common-config.yaml

```diff
@@ -0,0 +1,13 @@
+calicoAccountant:
+  backend: nftables
```

Comment:
Yes it is based on the OS version, I can do that. I wasn't sure on how we should handle that one, I was considering to add a prompt for it for when you run init, but we don't have that for any other config so that would maybe be strange.